### DISCUSSION TITLE
📣 POLL: Have you used sample code in selling-partner-api-samples  repository in your application?

### DISCUSSION THREAD

**Discussion by weilinggu** [amzn/selling-partner-api-models#4063] [Polls]
The SP-API team is looking for your feedback in order to identify the resources that would help you the most in your API integration journey.
Please answer the poll below, raise your voice, and help us shape the future of SP-API content!

**Comment by J-W-Chan** [comment]
no c# demo

**Comment by weilinggu** [comment]
You can find the sample code in [Amazon Selling Partner API Samples Repository](https://github.com/amzn/selling-partner-api-samples)

**Comment by johnkw** [comment]
I reviewed it to see if it had any hint on how to get past or workaround the blocker bug #556 for `FBA_INB_0182` but it did not.

**Comment by Ponera-Syl** [comment]
No because they are not in PHP
